tensorflow/tools/ci_build/Dockerfile.gpu

```diff
@@ -1,7 +1,12 @@
-FROM nvidia/cuda:8.0-cudnn5-devel-ubuntu16.04
+FROM nvidia/cuda:8.0-cudnn5-devel-ubuntu14.04
```

Comment:
Note that if we merge this in, tests will break for two reasons:
- The tests in master no longer work with Ubuntu 14.04. We have to restore the python 3.5 stuff.
- The tests are known to crash on Ubuntu 14.04 unless we install numpy from scratch. We have to update that too.

Delete for now?